packages/core/src/components/Menu/Menu/__stories__/Menu.stories.tsx

```diff
@@ -5,13 +5,23 @@ import { DialogContentContainer, Menu, MenuDivider, MenuGridItem, MenuItem, Menu
 import { DummyNavigableGrid } from "../../../GridKeyboardNavigationContext/__stories__/useGridKeyboardNavigationContext.stories.helpers";
 import { menuWithTwoDepthsSuite } from "../__tests__/Menu.interactions";
 import styles from "./Menu.stories.module.scss";
+import { MenuProps } from "../../..";
+import { SIZES } from "../../../../constants/sizes";
+
+import { VibeComponent } from "src/types";
 
 export default {
   title: "Navigation/Menu/Menu",
   component: Menu
 };
 
-const menuTemplate = args => (
+const menuTemplate = (
+  args: VibeComponent<MenuProps> & {
+    isMenu?: boolean;
+    supportFocusOnMount?: boolean;
+    sizes?: typeof SIZES;
+  }
```

Comment:
What if you just use `MenuProps` ? Will it still require these static props?

Reply:
It would not :)